src/main/java/org/example/homework7/Calculator.java

```diff
@@ -0,0 +1,26 @@
+package org.example.homework7;
+
+public class Calculator {
+
+    public int calculate(RomanNumber num1, RomanNumber num2, String operation) {
+
+        int Result = 0;
+
+        switch (operation) {
+            case "+" ->
+                Result = num1.getValue() + num2.getValue();
```

Comment:
It would be better to define `value1 = num1.getValue()` + and `value2 num2.getValue()` before the switch, and operate with `value1` and `value2` inside the switch